xtevent/xtevent.sthlp

```diff
@@ -192,13 +194,17 @@ trend (_ttrend) to the regression. {opt (gmm)} uses the GMM to compute the trend
 {opt saveov:erlay} saves estimations for the overlay plot produced by {opt xteventplot, overlay(trend)}.
 
 {phang}
-{opt savek(stub [, noestimate])} saves variables for event-time dummies, event-time, and trends. Event-time dummies are stored as {it: stub}_eq_m# for the dummy
-variable # periods before the policy change, and {it:stub}_p# for the dummy variable # periods after the policy change. The dummy variable for
-the policy change time is {it:stub}_p0. Event time is stored as {it:stub}_evtime. The trend is stored as {it:stub}_trend. The following suboption can be specified:
+{opt savek(stub [, subopt])} saves variables for time-to-event, event-time, trend, and interaction variables. Event-time dummies are stored as {it: stub}_eq_m# for the dummy
+variable # periods before the policy change, and {it:stub}_eq_p# for the dummy variable # periods after the policy change. The dummy variable for
+the policy change time is {it:stub}_eq_p0. Event time is stored as {it:stub}_evtime. The trend is stored as {it:stub}_trend. The interaction variables are stored as {it:stub}_m#_c# or {it:stub}_p#_c#, where c# indicates the cohort. The following 
+suboptions can be specified:
 
 {phang2}
 {opt noe:stimate} saves variables for event-time dummies, event-time and trends without estimating the model. This is useful if the users want to customize their regressions and plots.
 
+{phang2}
+{opt saveint:eract} saves interaction variables if {opt cohort} and {opt control_cohort} are specified. {opt noe:stimate} and {opt saveint:eract} cannot be specified simultaneously.
+

```

Comment:
@Constantino-Carreto-Romero Document that `avar` is needed for the SA options.

Reply:
I implemented it in https://github.com/JMSLab/xtevent/commit/c0e95d7073797f719a264370a5b700095fcb2785

Reply:
@Constantino-Carreto-Romero Some typos in options vs commands.

Reply:
I changed it in https://github.com/JMSLab/xtevent/commit/c9ae7ac9c4c0ef04ca984ea53c986a8d5cb85d9d